go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -450,28 +450,119 @@ func handleNetdevResources(resourceName string) (string, error) {
 	return netdevice, nil
 }
 
-func createNodeManagementPorts(name string, nodeAnnotator kube.Annotator, waiter *startupWaiter,
-	subnets []*net.IPNet) ([]managementPortEntry, *managementPortConfig, error) {
-	// If netdevice name is not provided in the full mode then management port backed by OVS internal port.
-	// If it is provided then it is backed by VF or SF and need to determine its representor name to plug
-	// into OVS integrational bridge
-	if config.OvnKubeNode.Mode == types.NodeModeFull && config.OvnKubeNode.MgmtPortNetdev != "" {
-		deviceID, err := util.GetDeviceIDFromNetdevice(config.OvnKubeNode.MgmtPortNetdev)
+func exportManagementPortAnnotation(netdevName string, nodeAnnotator kube.Annotator) error {
+	klog.Infof("Exporting management port annotation for netdev '%v'", netdevName)
+	deviceID, err := util.GetDeviceIDFromNetdevice(netdevName)
+	if err != nil {
+		return err
+	}
+	vfindex, err := util.GetSriovnetOps().GetVfIndexByPciAddress(deviceID)
+	if err != nil {
+		return err
+	}
+	pfindex, err := util.GetSriovnetOps().GetPfIndexByVfPciAddress(deviceID)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Optional: Maybe we can have a print like `klog.Infof("Exported management port annotation pfindex '%v' and vfindex '%v' for netdev '%v'", pfindex, vfindex netdevName)`

Reply:
I'd like to avoid too many prints. It's not free. It feels almost like a debug print. The annotations can be checked to see if the export properly happened.